DSL/Query/Span/SpanNearQuery.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\DSL\Query\Span;
+
+use ONGR\ElasticsearchBundle\DSL\ParametersTrait;
+
+/**
+ * Elasticsearch span near query.
+ */
+class SpanNearQuery implements SpanQueryInterface
+{
+    use ParametersTrait;
+
+    /**
+     * @var int
+     */
+    private $slop;
+
+    /**
+     * @var SpanQueryInterface[]
+     */
+    private $queries = [];
+
+    /**
+     * @param int                  $slop
+     * @param SpanQueryInterface[] $queries
+     * @param array                $parameters
+     *
+     * @throws \LogicException
+     */
+    public function __construct($slop, array $queries = [], array $parameters = [])
```

Comment:
Remove `$queries` here. Queries should be added via `addQuery(SpanQueryInterface $query)`.